pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -949,6 +954,22 @@ func (tc *TidbCluster) IsTiDBBinlogEnabled() bool {
 	return *binlogEnabled
 }
 
+func (tidb *TiDBSpec) IsBootstrapSQLEnabled() bool {
```

Comment:
How about do not assume which version support it, just checking `tidb.BootstrapSQLConfigMapName`?
let it fail to start if the version does not support but you set `BootstrapSQLConfigMapName`

Reply:
make sense to me